rbac/rbac/middleware.py

```diff
@@ -184,12 +195,14 @@ def process_request(self, request):  # pylint: disable=R1710
         except (KeyError, JSONDecodeError):
             request_psk = request.META.get(RH_RBAC_PSK)
             account = request.META.get(RH_RBAC_ACCOUNT)
+            org_id = request.META.get(RH_RBAC_ORG_ID)
             client_id = request.META.get(RH_RBAC_CLIENT_ID)
-            has_system_auth_headers = request_psk and account and client_id
+            has_system_auth_headers = request_psk and account and client_id and org_id
```

Comment:
Will this "and" break existing usage, when this org_id is not required yet?

Reply:
Yes, I got ahead of myself. It now will store it but NOT require org_id.